server/crawler/index.ts

```diff
@@ -12,136 +13,149 @@ function formatBody({
   author: string,
   description: string,
 }) {
-  const style = `
-<style>
-  html, body {
-    margin: 0;
-    padding: 0;
-  }
-  body {
-    font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", sans-serif;
-    font-size: 18px;
-    line-height: 1.5;
-  }
-  main {
-    max-width: 673px;
-    margin: 40px auto;
-    padding: 0 20px;
-  }
-  hr { height: 1px; }
-  h1, h2, h3, h4, h5, h6 { font-weight: 600; line-height: 1.4; }
-  h1 { font-size: 28px; }
-  h2 { font-size: 24px; }
-  h3 { font-size: 22px; }
-  h4 { font-size: 18px; }
-  h5 { font-size: 16px; }
-  h6 { font-size: 14px; }
-  li ul, li ol { margin: 0 20px; }
-  li { margin: 20px 0; }
-  ul { list-style-type: disc; }
-  ol { list-style-type: decimal; }
-  ol ol { list-style: upper-alpha; }
-  ol ol ol { list-style: lower-roman; }
-  ol ol ol ol { list-style: lower-alpha; }
-  img, video, audio {
-    display: block;
-    max-width: 100%;
-    margin: 0 auto;
-  }
-  audio {
-    width: 100%;
+  let meta = ''
+  if (description) {
+    meta += `
+    <meta name="description" content="${description}" />
+    <meta property="og:description" content="${description}">
+    <meta name="twitter:description" content="${description}">`
   }
-  blockquote {
-    margin-left: 20px;
-    margin-right: 20px;
-    color: #5F5F5F;
+  if (author) {
+    meta += `
+    <meta property="article:author" content="${author}">
+    <meta property="og:title" content="${author}: ${title}">
+    <meta name="twitter:title" content="${author}: ${title}">`
```

Comment:
If author not exists, title should just be `title`, without `og:title` we will have many preview display issue